### ISSUE TITLE
protobuf used as model

### ISSUE THREAD

**Issue by fabriziomoscon** [sybrexsys/csvmover#4]
I noticed that `*pb.UserInfoRequest` is used as model for your data.
Do you see any potential issue with using protobuf for modelling your data?

**Comment by sybrexsys** [comment]
Of course, I can write my personal model but it was the easy way to work. 
In case we will move to another communication engine this model can be rewritten
The biggest problem is using a lot of the fields what is used for the internal work of the gRPC
